vtex-sites/gatsby.store
# Chore: Improves rows and dividers on storybook

ported from https://github.com/vtex-sites/nextjs.store/pull/106

## What's the purpose of this pull request?

Attempts to improve the row styles. It looked a bit confusing because we were using alternate colors for the rows, and we also had some empty rows for the dividers.

**This change will only be applied to the docs already using `TokenDivider`.
We will be updating all the component's docs gradually.

## How does it work?

|Before|
|-|
![Before](https://user-images.githubusercontent.com/3356699/172447743-c5577a8d-0ede-4a68-baa1-85c5982a0eff.png)

|After|
|-|
![After](https://user-images.githubusercontent.com/3356699/172448495-2f7690f4-6928-4a93-bbe0-da3b5c7b3f00.png)

## How to test it?

Check on the [preview](https://gatsby-store-storybook-git-chore-improve-divid-ab642a-faststore.vercel.app/): all the rows should have white background color applied and the dividers, light gray background color.

**This change will only be applied to the docs already using `TokenDivider`.
We will be updating all the component's docs gradually.

## Checklist

<em>You may erase this after checking them all ;)</em>

**Changelog**

**PR Description**

**Documentation**
